cerebrium/cortex/introduction.mdx

```diff
@@ -67,18 +67,24 @@ cerebrium deploy <MODEL_NAME> <PRIVATE_API_KEY> --hardware=<HARDWARE>
 For the parameters above:
 
 - **\<MODEL_NAME\>**: Give your model a name for you to recognize on your Cerebrium dashboard
-- **\<HARDWARE\>**: This is either CPU, GPU or A10
 - **\<PRIVATE_API_KEY\>**: This is the private API key you can find on your Cerebrium dashboard
+- **\<HARDWARE\>**: This is either CPU, GPU or A10
+
+Additionally, we provide two flags to further control which files you add to your deployment. They are:
+
+- **--include**: Regrex string of files or folders to include. eg. `--include "./modelFiles/" --include my-awesome-file.txt`
```

Comment:
Why is include specified multiple times and exclude is an array with multiple names. If both are possible I think just do the array one - its easier

Reply:
Wrote it with just the array but couldn't find a way to demonstrate both options.
Nice suggestion, will change it to that if you think it's clearer!